src/util.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export function addDatabaseIdToRequest(datastore: any, reqOpts: any) {
```

Comment:
this looks like it's only used in one place in the (non test) code. Is this necessary, or can we simply inline it?

Reply:
I guess we could inline it, move this to `test/util.ts` so that multiple test files can use this shared code and then add a comment to say that the same logic is in two places.

Reply:
Second thought, I really think we should avoid having duplicate code in two places (this logic in the test code and this logic in request.ts). So I think we should keep this function here and call it directly from both places instead of going through `addDatabaseIdToRequest` on `index.ts`.

Reply:
Will add some more comments in tests specifically